app/utils/auth_helper.py

```diff
@@ -14,24 +14,15 @@
 def get_user_from_jwt(req, api_key):
     jwt_helper = JWT(api_key)
 
-    auth = req.headers.get('Authorization', None)
+    token = req.headers.get('Authorization', None)
+    if token is None:
+        token = req.headers.get('Auth-Token', None)
```

Comment:
Actually, scratch that. Authorization is the standard one so let's stick with that.